src/main/thrift/parquet.thrift

```diff
@@ -511,6 +511,51 @@ struct ColumnMetaData {
    * This information can be used to determine if all data pages are
    * dictionary encoded for example **/
   13: optional list<PageEncodingStats> encoding_stats;
+
+  /** Byte offset from beginning of file to bloom filter data**/
+  14: optional i64 bloom_filter_offset;
+}
+
+/**
+  * Definition of bloom filter algorithm.
+  */
+enum BloomFilterAlgorithm {
+  /** Block based bloom filter. 
+   * The bloom filter bitset is separated into tiny bucket as tiny bloom 
+   * filter, the high 32 bits hash value is used to select bucket, and 
+   * lower 32 bits hash values are used to set bits in tiny bloom filter.
+   * See “Cache-, Hash- and Space-Efficient Bloom Filters”. Specifically, 
+   * the tiny bloom filter size is 32 bytes and the algorithm set 8 bits
+   * in each bucket.
+   *
+   * In order to set bits in bucket, the algorithm need 8 SALT values 
+   * (0x47b6137bU, 0x44974d91U, 0x8824ad5bU, 0xa2b7289dU, 0x705495c7U, 
+   * 0x2df1424bU, 0x9efc4947U, 0x5c6bfb31U) to calculate index with formular:
+   *                  index[i] = (hash * SALT[i]) >> 27 
```

Comment:
This doesn't quite explain it, since index[i] will be a number between 0 and 31, inclusive, while the tiny bloom filter is 32 bytes, aka 256 bits. Please show that it is a split bloom filter over eight 32-bit words.